docs/getting-started.md

```diff
@@ -45,6 +45,8 @@ If you intend to start containerd via systemd, you should also download the `con
 https://raw.githubusercontent.com/containerd/containerd/main/containerd.service into `/usr/local/lib/systemd/system/containerd.service`,
 and run the following commands:
 
+> NOTE: For some flavors like Ubuntu, this will be `/usr/lib/systemd/system/containerd.service` instead.
+
```

Comment:
`/usr/local/lib/systemd/system/containerd.service` is correct for Ubuntu too.
Verified on Ubuntu 22.10 and 18.04.

```console
root@lima-ubuntu:~# cat /usr/local/lib/systemd/system/containerd.service 
# Copyright The containerd Authors.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

[Unit]
Description=containerd container runtime
Documentation=https://containerd.io
After=network.target local-fs.target

[Service]
ExecStartPre=-/sbin/modprobe overlay
ExecStart=/usr/local/bin/containerd

Type=notify
Delegate=yes
KillMode=process
Restart=always
RestartSec=5
# Having non-zero Limit*s causes performance problems due to accounting overhead
# in the kernel. We recommend using cgroups to do container-local accounting.
LimitNPROC=infinity
LimitCORE=infinity
LimitNOFILE=infinity
# Comment TasksMax if your systemd version does not supports it.
# Only systemd 226 and above support this version.
TasksMax=infinity
OOMScoreAdjust=-999

[Install]
WantedBy=multi-user.target

root@lima-ubuntu:~# systemctl enable --now containerd
Created symlink /etc/systemd/system/multi-user.target.wants/containerd.service → /usr/local/lib/systemd/system/containerd.service.

root@lima-ubuntu:~# nerdctl info
Client:
 Namespace:	default
 Debug Mode:	false

Server:
 Server Version: v1.6.12
 Storage Driver: overlayfs
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Cgroup Version: 1
 Plugins:
  Log: fluentd journald json-file syslog
  Storage: aufs native overlayfs
 Security Options:
  apparmor
  seccomp
   Profile: default
 Kernel Version: 4.15.0-202-generic
 Operating System: Ubuntu 18.04.6 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 4
 Total Memory: 3.846GiB
 Name: lima-ubuntu
 ID: 33138d20-a40e-4838-abea-dbbc35857412

WARNING: No swap limit support
WARNING: IPv4 forwarding is disabled
WARNING: bridge-nf-call-iptables is disabled
WARNING: bridge-nf-call-ip6tables is disabled
```

Reply:
```console
$ find /usr/local/lib/systemd/
/usr/local/lib/systemd/
/usr/local/lib/systemd/system
/usr/local/lib/systemd/system/containerd.service
/usr/local/lib/systemd/system/stargz-snapshotter.service
/usr/local/lib/systemd/system/buildkit.service
```

Reply:
It looks to me like perhaps your/my ubuntu's systemd was set up different? 🤔 

I'm running off a freshly installed VM on Hyper-V (Windows 11).

No `systemd` in `/usr/local/lib`
```
$ find /usr/local/lib
/usr/local/lib
/usr/local/lib/python3.10
/usr/local/lib/python3.10/dist-packages
```
Only in `/usr/lib/`.

My `nerdctl info`:

```
~$ sudo nerdctl info
Client:
 Namespace:	default
 Debug Mode:	false

Server:
 Server Version: v1.6.15
 Storage Driver: overlayfs
 Logging Driver: json-file
 Cgroup Driver: systemd
 Cgroup Version: 2
 Plugins:
  Log: fluentd journald json-file syslog
  Storage: native overlayfs
 Security Options:
  apparmor
  seccomp
   Profile: default
  cgroupns
 Kernel Version: 5.15.0-58-generic
 Operating System: Ubuntu 22.04.1 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 4
 Total Memory: 7.064GiB
 Name: nandaa-vm
 ID: d6d1e643-12fb-4421-8ed5-3b22f67074d2
 ```

Reply:
There could be something with my setup, we can hold on merging this one, until I find out what's happening. Any pointers will be appreciated.

Reply:
You can just mkdir it

Reply:
Sure, wanted to check if there was any relations with the issues I was getting downstream on the K8s side; but determined it was unrelated. 

I did `mkdir` and moved the one I'd put in `/usr/lib/systemd/system` to here.